### PR TITLE
AG-11137 Fix radial bar with negative values

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-11137

### CODE REVIEW

File: packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts [hunk 1]
```diff
@@ -261,21 +263,24 @@ export class RadialBarSeries extends _ModuleSupport.PolarSeries<
...
+            let rangeStartAngle = angleScale.convert(0, { clampMode: 'clamped' });
             let rangeEndAngle = angleScale.convert(angleRange, { clampMode: 'clamped' });
 
-            if (angleAxisReversed) {
+            if (!isPositive || angleAxisReversed) {
                 [rangeStartAngle, rangeEndAngle] = [rangeEndAngle, rangeStartAngle];
                 [startAngle, endAngle] = [endAngle, startAngle];
             }
 
+            console.log(angleRange, [startAngle, endAngle], [rangeStartAngle, rangeEndAngle]);
```
Comment (alantreadway) [inline packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts]: NIt: Remove `console.log`?

Comment (jacobp100) [inline packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts]:
```suggestion
```

Comment (jacobp100) [reply packages/ag-charts-enterprise/src/series/radial-bar/radialBarSeries.ts]: 🤦‍♂️

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**